app.py

```diff
@@ -55,24 +55,38 @@ def experience():
     json
         The experience(s) as a JSON object, or an error message.
     '''
+    response = None
```

Comment:
This is an interesting refactor.

On one hand I agree that having this response object makes it clear that we are just returning a single response at a time.

One the other, I kinda think that having multiple return statements make the flow look cleaner. I'm trying to think from a new reader POV, that having a return statement will easily finish the code flow from that point, making it easy to follow through.

But, I am not 100% on either side, maybe this is more of a personal style. What do you think we should do here?

Reply:
What do you think of this Satoshi? We can keep it the way it is now and the way I mentioned as well. I'm happy with either but just wanted to point out both these options. What do you say?